packages/block-library/src/navigation/index.php

```diff
@@ -341,7 +341,11 @@ function block_core_navigation_get_fallback_blocks() {
 	// If `core/page-list` is not registered then return empty blocks.
 	$fallback_blocks = $registry->is_registered( 'core/page-list' ) ? $page_list_fallback : array();
 
-	$navigation_post = Gutenberg_Navigation_Fallback::get_fallback();
+	if ( class_exists( 'WP_Navigation_Fallback' ) ) {
```

Comment:
Not sure if I this should be

```php
if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
```

Same effect, just convention?

Reply:
I'd go with the `class_exists` check, since the compat layer changes over time (and is thus not directly equivalent to the presence of the plugin).